app/assets/javascripts/modules/sticky-element-container.js

```diff
@@ -10,77 +10,77 @@
 (function (Modules, root) {
   'use strict'
 
-  var $ = root.$
-  var $window = $(root)
-
   Modules.StickyElementContainer = function () {
     var self = this
 
-    self._getWindowDimensions = function _getWindowDimensions () {
+    self.getWindowDimensions = function () {
```

Comment:
I think it would be good to investigating converting this to using the more standard module format used by the rest of our JS, e.g.

```
window.GOVUK = window.GOVUK || {}
window.GOVUK.Modules = window.GOVUK.Modules || {};

(function (Modules) {
  function StickyElementContainer ($module) {
    this.module = $module
  }

  StickyElementContainer.prototype.init = function () {
    // etc.
  }

Modules.StickyElementContainer = StickyElementContainer
})(window.GOVUK.Modules)
```

However I don't think this should block this PR in its current form - we should raise a separate PR to do this afterwards.

Reply:
We could also look at improving the readability of this code (not related to your changes). For example, renaming `window` to `root` caused me some initial confusion, as well as what the initialise function is actually doing and the names of the elements involved (`el` and `element` could be better as `wrapper` and `stickyElement`, for example).

Reply:
I tried to make this into a module 😢 - my skills are lacking and I hit a blocker maintaining a reference to the window object in the event handlers. Happy to give it a go, but might need some assist with that aspect.

Yeah, the naming is not great and totally agree with your suggestions.  I'll change the naming commit to include them.

Reply:
Raised [this card](https://trello.com/c/XwULsSw8/316-convert-stick-element-in-government-frontend-to-follow-the-module-pattern) to address this issue.